eng/azure-pipelines-codeql.yml

```diff
@@ -0,0 +1,101 @@
+trigger:
+  none
+
+schedules:
+  - cron: 0 12 * * 1
+    displayName: Weekly Monday CodeQL/Semmle run
+    branches:
+      include:
+      - main
```

Comment:
The branch is called `dotnet/main` here in this repo. 

I now wonder whether we missed this in other repos that use `dotnet/main` instead of `main`, can you please check?

Reply:
Ahh thanks for the catch... I'll go through them all and make PRs where I got the branch wrong.

Reply:
Looks like out of all the repos that we had enabled CodeQL for, only this one has the default branch as `dotnet/main` instead of `main` 😌